Lib/test/test_subprocess.py

```diff
@@ -59,6 +59,8 @@ def setUp(self):
         support.reap_children()
 
     def tearDown(self):
+        if mswindows:
```

Comment:
I don't care so much about whether we early-exit from `tearDown` or just don't define it, but it's probably easier to read the code if we skip the definition completely. It might also help prevent people from adding code that _does_ matter on Windows at the end of the function without noticing that it's not going to run.